cli/serve/static/scan.html

```diff
@@ -0,0 +1,106 @@
+
+  <title>CFSSL - Scan</title>
+  <link href='//cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.4/css/bootstrap.min.css' rel='stylesheet prefetch'>
+  <style>
+  body {
+      padding-top: 30px;
+    }
+  </style>
+</head>
+
+<body>
+  <nav class="navbar navbar-default navbar-fixed-top">
+    <div class="container">
+      <div class="navbar-header">
+        <a class="navbar-brand" href="/">CFSSL</a>
+      </div>
+
+      <div class="collapse navbar-collapse">
+        <ul class="nav navbar-nav">
+          <li>
+            <a href="/scan">Scan</a>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </nav>
+
+  <div class="container">
+    <h1 class="page-header">Scan</h1>
+
+    <div class="row">
+      <div class="col-sm-6 col-md-4 col-md-offset-4">
+        <form class="form-inline" id="cfssl-scan" name="cfssl-scan">
+          <label for="host">Host</label> <input class="form-control" id="host" name="host" placeholder="cfssl.org" type="text"> <button class="btn btn-primary" type="submit">Scan</button>
+        </form>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="col-sm-6 col-md-4 col-md-offset-4">
+        <table class="table table-bordered table-striped" id="results"></table>
+
+        <div class="panel-group" id="accordion"></div>
+      </div>
+    </div>
+  </div>
+
+  <script src="//cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+  <script>
+    $('#cfssl-scan').submit(function(event) {
+      $('#results').empty();
+      $.getJSON('/api/v1/cfssl/scan?host=' + $('#host').val(), function(data) {
+        if (data['success'] && data['result']) {
+          $.each(data['result'], function(family_name, scans) {
+            var result_row = '<tr>';
+            result_row += '<td>' + family_name + '<\/td>';
+            result_row += '<td><table class="table table-bordered table-striped">';
+            $.each(scans, function(scanner_name, result) {
+              result_row += '<tr>'
+              result_row += '<td>' + scanner_name + '<\/td>'
+              var grade = result["grade"];
+              if (grade == "Good") {
+                result_row += '<td><span class="glyphicon glyphicon-ok-sign"><\/span> Good<\/td>';
+              } else if (grade == "Warning") {
+                result_row += '<td><span class="glyphicon glyphicon-exclamation-sign"><\/span> Warning<\/td>';
+              } else if (grade == "Bad") {
+                result_row += '<td><span class="glyphicon glyphicon-remove-sign"><\/span> Bad<\/td>';
+              } else {
+                result_row += '<td><span class="glyphicon glyphicon-question-sign"><\/span><\/td>';
+              }
+              result_row += '<\/tr>';
+              var output = result['output'];
+              if (output !== undefined) {
+                result_row += '<td>';
+                var output_type = $.type(output);
+                if (output_type == "object" || output_type == "array") {
```

Comment:
You should consistently use single quotes (except for HTML attributes inside of a string).